src/modules/jobs/controllers/jobs.controller.ts

```diff
@@ -1,14 +1,26 @@
 import { Controller, Get, Param } from '@nestjs/common';
 
+import { TechLoad } from '../models/tech-load.model';
+
 @Controller('/jobs')
 export class JobsController {
-  @Get('/get/:uuid')
-  async getJobs(@Param('uuid') uuid: string) {
-    await new Promise(resolve => setTimeout(resolve, 1000));
+  @Get(['get/:uuid', '/get/:uuid/:amount'])
+  async getJobsSpecific(@Param('uuid') uuid: string, @Param('amount') amount: string) {
+    await new Promise(resolve => setTimeout(resolve, 1500)); // just adds a delay for feel
+
+    if (!uuid || uuid.length === 0) {
+      return {
+        'flowStatus': 'FAILURE',
+        'flowStatusMessage': 'invalid UUID',
```

Comment:
Return message is misleading. Says invalid uuid though the if statement is accounting for a lack of uuid...

I think you should add proper uuid formatting as the invalid option.

Reply:
It is done. Made a little better as well.